.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_API_KEY=AIzaSyDcHCwb7QJaeUbgP6QblmJFqh2SZMQTzFU
+REACT_APP_AUTH_DOMAIN=fir-local-test-e5454.firebaseapp.com
+REACT_APP_PROJECT_ID=fir-local-test-e5454
+REACT_APP_STORAGE_BUCKET=fir-local-test-e5454.appspot.com
+REACT_APP_MESSAGING_SENDER_ID=253058253425
+REACT_APP_MEASUREMENT_ID=G-3Q5827W2EP
```

Comment:
I don't think we want this? This should be a `.env.local` file?

Reply:
It's now been deleted